apps/tangle-dapp/components/LiquidStaking/unstakeRequestsTable/UnstakeRequestsTable.tsx

```diff
@@ -92,9 +93,7 @@ const columns = [
           return undefined;
         }
 
-        const plurality = remainingTimeUnit.value === 1 ? '' : 's';
-
-        return `${remainingTimeUnit.value} ${remainingTimeUnit.unit}${plurality}`;
+        return stringifyTimeUnit(remainingTimeUnit);
```

Comment:
Returning a tuple allows for concatenating strings with whitespace in between, which I believe is safer.